src/frontend/src/app/lobby/lobby.component.ts

```diff
@@ -43,4 +46,12 @@ export class LobbyComponent implements OnInit {
   startGame() {
     this.ws.nextRound();
   }
+
+  copyCode() {
+    this.clipboard.copy(this.id);
+  }
+
+  leave() {
+    this.router.navigate(['/']);
```

Comment:
I think you also have to clear the lobbyCode in the websocket service. Otherwise its still in the lobby.